src/lib/containers/download_list_v2/CreatePackageV2.tsx

```diff
@@ -28,6 +30,9 @@ export const CreatePackageV2 = (props: CreatePackageV2Props) => {
   const { accessToken } = useSynapseContext()
   const [isLoading, setIsLoading] = useState(false)
   const [fileName, setZipFileName] = useState('')
+  const [isIncludingManifest, setIsIncludingManifest] = useState(true)
```

Comment:
Option for the user to create/download the manifest file for the entire list (if iteratively downloading files, this option will only be available on the initial entry into the component).